actions/followUser.ts

```diff
@@ -0,0 +1,24 @@
+'use server';
+import { auth } from '@/app/api/auth/[...nextauth]/auth';
+import mumbleUserServiceInstance from '@/services/Mumble/MumbleUser';
+import { revalidatePath } from 'next/cache';
+
+export const followUserToggle = async ({
+  identifier,
+  unfollow = false,
+}: {
+  identifier: string;
```

Comment:
an interface for those types would be good. as we did in other components / functions